connectors/src/connectors/zendesk/temporal/workflows.ts

```diff
@@ -33,6 +29,14 @@ const {
   startToCloseTimeout: "1 minute",
 });
 
+function assertNever(x: never): never {
```

Comment:
We likely already have it available in connectors no?

Reply:
my bad for not checking, we actually don't but temporal does. Using it